api/v1beta1/glanceapi_types.go

```diff
@@ -115,6 +115,9 @@ type GlanceAPIStatus struct {
 	// then the controller has not processed the latest changes injected by
 	// the opentack-operator in the top-level CR (e.g. the ContainerImage)
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+
+	// LastAppliedTopology - the last applied Topology
+	LastAppliedTopology string `json:"lastAppliedTopology,omitempty"`
```

Comment:
should we also keep track of the `LastAppliedTopologyGeneration`, or is that detail of tracking to much?

Reply:
No we don't, `LastAppliedTopology` is basically enough to "try" to remove a finalizer if is part of the list, otherwise it's skipped. I think this is enough to make sure we free resources in case we delete the `topologyRef` key from the CR.